### ISSUE TITLE
Pivot component: Selected tab text is not visible in high contrast mode

### ISSUE THREAD

**Issue by shaikshavalis** [microsoft/fluentui#15075]
### Environment Information

- **Package version(s)**:NA, can repro in latest version
- **Browser and OS versions**:  Edge Chromium, Windows

### Describe the issue:
Selected tab text is not visible clearly in high contrast mode on new edge browser.

### Please provide a reproduction of the issue in a codepen:

can be seen on codepens available in documentation.

#### Actual behavior:
![image](https://user-images.githubusercontent.com/9265381/93385424-e2efec00-f883-11ea-93f3-7b3b374c48a5.png)
 **without high contrast mode**
![image](https://user-images.githubusercontent.com/9265381/93385569-0fa40380-f884-11ea-9006-df9ece49438f.png)

#### Expected behavior:
selected tab text should be visible based on selected high contrast theme.

**Comment by paulgildea** [comment]
@shaikshavalis Thanks for the feedback.

@smhigley FYI
@xugao Heads up as I know you were doing some HC work lately.

**Comment by smhigley** [comment]
I haven't investigated really, but my initial wild guess would be to look at the `::before` pseudo-element

**Comment by pritini** [comment]
@smhigley 👍 
The color attribute "HighlightText" is causing this!
